cltk/corpus/gujarati/alphabet.py

```diff
@@ -0,0 +1,19 @@
+""" GUJARATI is an INDO-ARYAN language native to the state of Gujarat. It is spoken in the state of Gujarat in addition to the
+union territories of Daman & Diu and Dadra & Nagar Haveli.
+
+The Gujarati language has 44 letters 11 VOWELS and 33 CONSONANTS.
+"""
+
+__author__ = ['Dhruv Apte']
+__license__ = 'MIT License.See LICENSE.'
+
+DIGITS = ['૧','૨','૩','૪','૫','૬','૭','૮','૯','૧૦']
+
+VOWELS = [ 'અ' , 'આ' , 'ઇ' , 'ઈ' , 'ઉ' , 'ઊ' , 'ઋ' , 'એ' , 'ઐ' , 'ઓ' , 'ઔ' ] 
```

Comment:
Through my quick google search i found Gujarati language has  13 vowels `અ આ ઇ ઈ ઉ ઊ ઋ એ ઐ ઓ ઔ અઁ અ:` divided in 2 sections short/long vowels, Please cloud you check it ?